js/varien/js.js

```diff
@@ -703,3 +703,50 @@ function buttonDisabler() {
         button.disabled = true;
     });
 }
+
+/**
+ * Adds copy icons to elements with the class 'copy-text'.
+ */
+function addCopyIcons() {
+    if (navigator.clipboard === undefined) {
+        return;
+    }
+
+    const copyTexts = document.querySelectorAll('.copy-text');
+    copyTexts.forEach(copyText => {
+        const iconStyle = JSON.parse(copyText.getAttribute('data-copy-icon'));
+        const svg = createSVGElement(iconStyle);
+        copyText.parentNode.appendChild(svg);
+    });
+}
+
+/**
+ * Creates an SVG element with the specified iconStyle.
+ *
+ * @param {Object} iconStyles - An object containing the style properties for the SVG icon.
+ * @param {string} [iconStyles.cursor='pointer'] - The cursor style for the SVG element.
+ * @param {string} iconStyles.height - The height of the SVG element.
+ * @param {string} iconStyles.width - The width of the SVG element.
+ * @param {string} [iconStyles.margin='0'] - The margin of the SVG element.
+ * @return {HTMLElement} The created SVG element.
+ */
+function createSVGElement(iconStyles) {
```

Comment:
better to rename this function to a more specific task name
```suggestion
function createIconCopyElement(iconStyles) {
```

Reply:
done, with a slightly different name :-)